### PR TITLE
Ensure that new deployments use the right OS distribution

### PR DESCRIPTION
**Pull request by ashb**
## Description

> Describe the purpose of this pull request.

## 🎟 Issue(s)

Resolves astronomer/issues#2100

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [ ] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `yarn lint && yarn test` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-UI](https://github.com/astronomer/astro-ui/) and [Astro-CLI](https://github.com/astronomer/astro-cli/) clients.
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: src/lib/deployments/config/index.js [hunk 1]
```diff
@@ -735,12 +735,20 @@ export async function defaultAirflowImage() {
...
 export async function airflowImageForVersion(version) {
-  return first(
-    filter(
-      await airflowImages(),
-      i =>
-        i.channel === "stable" && semver.coerce(i.version).version === version
-    )
+  const releasesOfVersion = filter(
+    await airflowImages(),
+    i => i.version == version
```
Comment (adam2k) [inline src/lib/deployments/config/index.js:740]:
```suggestion
    i => i.version === version
```

**Review by adam2k — APPROVED** [review]
LGTM if you can make that `===` change.

**Comment by adam2k** [comment]
Is there an issue and description for this issue too?

**Comment by ashb** [comment]
Not sure if an issue got created -- it was reported in Slack.

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**